src/core/components/AuthenticateButton/index.js

```diff
@@ -60,8 +60,18 @@ export class AuthenticateButtonBase extends React.Component {
     const buttonText = isAuthenticated ?
       logOutText || i18n.gettext('Log out') :
       logInText || i18n.gettext('Log in/Sign up');
+
+    // The `href` is required because a <button> element with a :hover effect
+    // and/or focus effect (that is not part of a form) that changes its
+    // appearance will transition to the hover state onClick when using a
+    // mobile browser. This is the cause of
+    // https://github.com/mozilla/addons-frontend/issues/1904
+    //
+    // We tried to use https://css-tricks.com/annoying-mobile-double-tap-link-issue/#article-header-id-2
```

Comment:
I don't think you need to include this last sentence because it's more relevant to the development process and not the code itself. Keeping comments concise always helps since other devs don't always read comments.